SimpleWAWS/Code/TemplatesManager.cs

```diff
@@ -60,17 +65,18 @@ static TemplatesManager()
                     Name = "Ping Site",
                     SpriteName = "sprite-PingSite PingSite",
                     AppService = AppService.Logic,
-                    CsmTemplateFilePath = HostingEnvironment.MapPath("~/CSMTemplates/PingSite.json"),
+                    CsmTemplateFilePath = HostingEnvironment.MapPath("~/ARMTemplates/PingSite.json"),
                     Description = Resources.Server.Templates_PingSiteDescription
                 });
                 list.Add(new JenkinsTemplate
                 {
                     Name = "Jenkins CI",
                     SpriteName = "sprite-PingSite PingSite",
                     AppService = AppService.Jenkins,
-                    CsmTemplateFilePath = HostingEnvironment.MapPath("~/CSMTemplates/JenkinsResource.json"),
+                    CsmTemplateFilePath = HostingEnvironment.MapPath("~/ARMTemplates/JenkinsResource.json"),
                     Description = Resources.Server.Templates_JenkinsDescription
                 });
+                _baseARMTemplate = JObject.Parse( File.ReadAllText(HostingEnvironment.MapPath("~/ARMTemplates/BaseARMTemplate.json")));
```

Comment:
Formatting.
Also can you add a comment explaining what this is and how it's used? It's not common to use JObject.FromObject to do a deep cloning of an object.